api/lib/resolvers.js

```diff
@@ -584,8 +573,10 @@ const resolvers = {
       if (!remoteFetch(dataSources, networkId).getAddressRole) return undefined
 
       return await remoteFetch(dataSources, networkId).getAddressRole(address)
-    },
-    storeUser: storeUser
+    }
+  },
+  Mutation: {
+    registerUser: registerUser
```

Comment:
what is the advantage of setting this as a mutation?

Reply:
over a Query I mean

Reply:
just conceptional: a query delivers data, a mutation manipulates data

Reply:
Oh I see